src/Controller/Organization/MissionType/MissionTypeDeleteController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization\MissionType;
+
+use App\Entity\MissionType;
+use App\Entity\Organization;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * @Route("/delete/{id}", name="app_organization_mission_type_delete", methods={"GET"})
+ */
+class MissionTypeDeleteController extends AbstractController
+{
+    public function __invoke(EntityManagerInterface $entityManager, MissionType $missionType): RedirectResponse
```

Comment:
please use the @ security tag to protect the controller

Reply:
yes, update coming soon ;)